packages/react-error-overlay/package.json

```diff
@@ -54,6 +56,7 @@
     "flow-bin": "^0.54.0",
     "jest": "20.0.4",
     "jest-fetch-mock": "1.2.1",
+    "raw-loader": "^0.5.1",
```

Comment:
Testing with the release by @react-scripts-dangerous caught this:
```bash
Failed to compile.

./node_modules/react-error-overlay-dangerous/lib/index.js
Module not found: Can't resolve 'raw-loader' in '/Users/joe/Desktop/folder2/node_modules/react-error-overlay-dangerous/lib'
```

I assume this should be a normal dependency.

Reply:
I assume our e2e tests didn't test this because we don't run npm with `NODE_ENV` as `production`; but can we do that in our tests -- I assume linking makes this really weird?

Is it worth trying to test in our E2E; or we could make @react-scripts-dangerous also run its own smoke tests and comment on PRs? Not sure how often this will be a problem ...

Reply:
Good catch with @react-scripts-dangerous 😀
Maybe in e2e, so that we can find errors before creating a PR??